packages/menu-bar/test/sub-menu.test.js

```diff
@@ -51,12 +51,6 @@ describe('sub-menu', () => {
     buttons = menu._buttons;
   });
 
-  afterEach(() => {
-    if (subMenu && subMenu.opened) {
-      subMenu.close();
```

Comment:
These blocks aren't needed because `vaadin-context-menu` closes itself on detach:

https://github.com/vaadin/web-components/blob/b7f250c8e32381b5c77c1f23f11aacf07a3ae0a3/packages/context-menu/src/vaadin-context-menu.js#L366-L371